src/api/vdom.js

```diff
@@ -44,17 +44,22 @@ function applyEvent (elem, ename, newFunc) {
     events = elem[$currentEventHandlers] = {};
   }
 
-  const oldFunc = events[ename];
-
-  // Remove old listener so they don't double up.
-  if (oldFunc) {
-    elem.removeEventListener(ename, oldFunc);
+  // Undefined indicates that there is no listener yet.
+  if (events[ename] === undefined) {
+    // We only add a single listener once. Originally this was a workaround for
```

Comment:
The original issue is this one https://github.com/webcomponents/shadydom/issues/24. In that test we were adding the same callback more than once, thus they didn't get removed.